Library/Formula/libodb-qt.rb

```diff
@@ -0,0 +1,78 @@
+class LibodbQt < Formula
+  homepage "http://www.codesynthesis.com/products/odb/"
+  url "http://www.codesynthesis.com/download/odb/2.4/libodb-qt-2.4.0.tar.gz"
+  sha256 "f48c563f653df178866301e4c5069b09b697d98f54dcc9ef8714157002156183"
+
+  option "without-qt", "Don't install the Qt library"
+  option "with-qt5", "Install version 5 of the framework instead of 4."
+
+  depends_on "libodb"
+  if build.without?("qt")
+  elsif build.with?("qt5")
+    depends_on "qt5"
+  else
+    depends_on "qt"
+  end
+
+  def install
+    qmake = "#{ENV["QMAKE"]}"
+
+    if qmake.empty?
+      qmake = `which qmake`.strip
+      if qmake.empty?
+        onoe "    Qt is a dependency, but you chose \"--without-qt\".
+             You need to indicate \`qmake\` path to homebrew as:
+             QMAKE=~/Qt/5.4/clang_64/bin/qmake brew install libodb-qt".undent
+        exit 1
+      end
+    end
```

Comment:
I have, `qt` (4) is set by default as a requirement, but for someone who already have installed the framework, why not letting him specify his `Qt`'s installation if he doesn't want to build with homebrew's `qt`?

Many `configure` scripts allow the user to specify the required libs with options like `--with-libcutl=`, but as `homebrew` doesn't follow the unrecognized options to `./configure`, if you don't allow to specify another path, then we must install homebrew's `qt`.

After that, if the user still have his installation of Qt, they may be not binary compatible, so if he compiles his app, it will be linked to Qt outside `homebrew` and `libodb-qt` which is linked to homebrew's `qt`. It's not practical, is it ?